jOOQ/src/main/java/org/jooq/Log.java

```diff
@@ -229,6 +239,84 @@
      */
     void error(Object message, Object details, Throwable throwable);
 
+    default boolean isEnabled(final Log.Level level) {
+        switch (level) {
+            case TRACE: return isTraceEnabled();
+            case DEBUG: return isDebugEnabled();
+            case INFO: return isInfoEnabled();
+            case WARN: return isWarnEnabled();
+            case ERROR: return isErrorEnabled();
+            case FATAL: return isErrorEnabled();
+            default: return false;
+        }
+    }
+
+    default void log(final Object message, final Log.Level level) {
```

Comment:
I'll reverse the order of arguments here. Seems more consistent to always have the level at the beginning.